website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -1,500 +1,102 @@
 ---
-title: Golang Based Operator Quickstart
-linkTitle: Quickstart
-weight: 10
+title: Quickstart for Go-based Operators
+linkTitle: QuickStart
+weight: 20
+description: A simple set of instructions that demonstrates the basics of setting up and running a Go-based operator.
 ---
 
-**NOTE:** For the SDK versions prior to `v0.19.0` please consult the [legacy docs][legacy_quickstart_doc] for the [legacy CLI][legacy_CLI] and project.
-
-This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
+This guide walks through an example of building a simple memcached-operator powered by Go using tools and libraries provided by the Operator SDK
 
 ## Prerequisites
 
-- [go][go_tool] version v1.13+.
-- [docker][docker_tool] version 17.03+.
-- [kubectl][kubectl_tool] version v1.11.3+.
-- [kustomize][kustomize_tool] v3.1.0+
-- Access to a Kubernetes v1.11.3+ cluster.
+- [Install operator-sdk][operator_install] and its prequisites.
+- Access to a Kubernetes v1.11.3+ cluster (v1.16.0+ if using `apiextensions.k8s.io/v1` CRDs).
 
-## Create a new project
+## Quickstart Steps
 
-Use the CLI to create a new memcached-operator project:
+### Create a project
 
+Create and change into a directory for your project. Then call `operator-sdk init`
+with the Go plugin to initialize the [base project layout][project_layout]:
+ 
 ```sh
-$ mkdir -p $HOME/projects/memcached-operator
-$ cd $HOME/projects/memcached-operator
-# we'll use a domain of example.com
-# so all API groups will be <group>.example.com
-$ operator-sdk init --domain=example.com --repo=github.com/example-inc/memcached-operator
-```
-
-To learn about the project directory structure, see [Kubebuilder project layout][kubebuilder_layout_doc] doc.
-
-#### A note on dependency management
-
-`operator-sdk init` generates a `go.mod` file to be used with [Go modules][go_mod_wiki]. The `--repo=<path>` flag is required when creating a project outside of `$GOPATH/src`, as scaffolded files require a valid module path. Ensure you [activate module support][activate_modules] by running `export GO111MODULE=on` before using the SDK.
-
-### Manager
-The main program for the operator `main.go` initializes and runs the [Manager][manager_go_doc].
-
-See the [Kubebuilder entrypoint doc][kubebuilder_entrypoint_doc] for more details on how the manager registers the Scheme for the custom resource API defintions, and sets up and runs controllers and webhooks.
-
-
-The Manager can restrict the namespace that all controllers will watch for resources:
-```Go
-mgr, err := manager.New(cfg, manager.Options{Namespace: namespace})
+mkdir memcached-operator
+cd memcached-operator
+operator-sdk init --repo=github.com/example-inc/memcached-operator
```

Comment:
```suggestion
operator-sdk init --domain=example.com --repo=github.com/example-inc/memcached-operator
```

Reply:
the domain is default ` --domain=example.com`. So, do we really need to add it in the quick start?

Reply:
The default is [`my.domain`](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugin/v3/init.go#L93).

Reply:
I think this is an important flag to include in the quickstart since pretty much _everyone_ will want to override this for any serious project.